crates/wit-parser/src/resolve.rs

```diff
@@ -2040,7 +2124,21 @@ impl Remap {
             .zip(&spans.includes)
             .zip(&include_names)
         {
-            if !resolve.include_stability(&stability) {
+            if !resolve
+                .include_stability(&stability, pkg_id)
+                .with_context(|| {
+                    format!(
+                        "failed to process feature gate for included world [{}] in package [{}]",
+                        resolve
+                            .worlds
+                            .get(include_world)
```

Comment:
It's ok to not use `.get` here since the index of `include_world` should always be valid, so indexing `[...]` syntax can be used

Reply:
Done!